python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -4773,6 +4773,75 @@ def test_rowwise_ops_nullable_int_dtypes(op, expected):
     assert_eq(got, expected)
 
 
+@pytest.mark.parametrize(
+    "data",
+    [
+        {
+            "t1": gd.Series(
+                ["2020-08-01 09:00:00", "1920-05-01 10:30:00"], dtype="<M8[ms]"
+            ),
+            "t2": gd.Series(
+                ["1940-08-31 06:00:00", "2020-08-02 10:00:00"], dtype="<M8[ms]"
+            ),
+        },
+        {
+            "t1": gd.Series(
+                ["2020-08-01 09:00:00", "1920-05-01 10:30:00"], dtype="<M8[ms]"
+            ),
+            "t2": gd.Series(
+                ["1940-08-31 06:00:00", "2020-08-02 10:00:00"], dtype="<M8[ns]"
+            ),
+            "t3": gd.Series(
+                ["1960-08-31 06:00:00", "2030-08-02 10:00:00"], dtype="<M8[s]"
+            ),
+        },
+        {
+            "t1": gd.Series(
+                ["2020-08-01 09:00:00", "1920-05-01 10:30:00"], dtype="<M8[ms]"
+            ),
+            "t2": gd.Series(
+                ["1940-08-31 06:00:00", "2020-08-02 10:00:00"], dtype="<M8[us]"
+            ),
+        },
+        {
+            "t1": gd.Series(
+                ["2020-08-01 09:00:00", "1920-05-01 10:30:00"], dtype="<M8[ms]"
+            ),
+            "t2": gd.Series(
+                ["1940-08-31 06:00:00", "2020-08-02 10:00:00"], dtype="<M8[ms]"
+            ),
+            "i1": gd.Series([1001, 2002], dtype="int64"),
+        },
+        {
+            "t1": ["2020-08-01 09:00:00", "1920-05-01 10:30:00"],
+            "t2": ["1940-08-31 06:00:00", pd.NaT],
+        },
```

Comment:
This test will not pass due to pandas bug (?)
https://github.com/pandas-dev/pandas/issues/36907#issue-715272559